content/id/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Menggunakan Port Forwarding untuk Mengakses Aplikasi di sebuah Klaster
+content_template: templates/task
+weight: 40
+min-kubernetes-server-version: v1.10
+---
+
+{{% capture overview %}}
+
+Halaman ini menunjukkan bagaimana menggunakan `kubectl port-forward` untuk menghubungkan sebuah server Redis yang sedang berjalan di sebuah klaster Kubernetes. Tipe dari koneksi ini dapat berguna untuk melakukan _debugging_ basis data.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Install [redis-cli](http://redis.io/topics/rediscli).
+
+{{% /capture %}}
+
+
+{{% capture steps %}}
+
+## Membuat deployment dan service Redis
+
+1. Buat sebuah Deployment yang menjalankan Redis:
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/application/guestbook/redis-master-deployment.yaml
+    ```
+
+    Keluaran dari sebuah perintah yang sukses memverifikasi bahwa Deployment telah terbuat:
```

Comment:
```suggestion
    Keluaran dari sebuah perintah yang sukses akan memverifikasi bahwa Deployment telah terbuat:
```
it's quite hard for me to understand the sentence, so I suggest to put `akan` or `,` between `sukses` and `memverifikasi`.
wdyt?

Reply:
putting `akan` seems nice to me